app/components/live-blog/live-blog-entries/live-blog-entries.less

```diff
@@ -57,11 +61,11 @@ live-blog-entries {
   .input-container {
     .mixin-input-container();
 
-    label {
+    label:not(.autocomplete-basic-label) {
```

Comment:
This is ugly, but short of refactoring all the styling for `autocomplete-basic` this is the best way to do this.